.gitignore

```diff
@@ -21,5 +21,8 @@ package-lock.json
 *.tgz
 
 # Repository specific
-coverage
+.cache
+.castor*
+examples/*/dist
```

Comment:
💅🏻  [nitpick] I believe a simple `dist` covers any folder by that name anywhere.